src/customRules/trailingCommaInterfaceRule.ts

```diff
@@ -0,0 +1,33 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+
+export class Rule extends Lint.Rules.AbstractRule {
+
+     /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
+        'ruleName': 'trailing-comma-interface',
+        'description': 'Enforce trailing commas in interface properties.',
+        'rationale': 'Our team uses traling commas in interfaces.',
```

Comment:
Which subsequently is the result of [this](https://github.com/Shopify/javascript#3.3), correct?

```
Why? Trailing commas allow you to add and remove a given property of an object without also editing the surrounding lines, which keeps the change isolated to the property in question.
```

But I don't think this is the same case for Typescript interfaces, since every Typescript interface property declarations needs a delimiter (`;` or `,`) on every line. Editing these lines won't interfere the surrounding lines in Git, unlike JS key-value pairs with missing trailing commas on the last key-value pair.

So wouldn't it be a team preference then?